core/tabs/tabs.py

```diff
@@ -48,3 +50,25 @@ def tab_move_right():
     # def tab_duplicate():
 
     # def tab_back():
+
+    # requires: https://github.com/gcmt/taboo.vim/
+    def tab_rename(text: str = ""):
+        actions.user.vim_run_command_exterm(f":TabooRename {text}")
+
+    def tab_new_named(text: str = ""):
+        actions.user.vim_run_command_exterm(f":TabooOpen {text}")
+
+    def tab_reset():
+        actions.user.vim_run_command_exterm(":TabooReset")
+
+
+@mod.action_class
+class Actions:
+    def tab_rename(text: str = ""):
```

Comment:
I think the tab_rename I added in my PR for new actions, and the PR to community didn't add the name text, so will have to be sure to sync that. It should have the name imo, so this more of a change for me but if we out this in it will break if the tab PR is fixed and merged I guess

Reply:
I did update that in the community PR, but forgot to resolve this